readme_en.md

```diff
@@ -116,6 +118,62 @@ For example, in the case of the following arrangement in Suxen, the wiring will
 - If you solder the Promicro to the "house" first, the solder may leak from the hole when wiring to the infinite possibilities side, and the parts of the Promicro body may be short-circuited.
      Be sure to solder the Promicro last.
 
+### Ordering PCBs
+
+Remember that the the manufacturers can change their requirements at any time!
+
+The basic process is that you want to submit the 'gerber' files in a zip.  In this repo, under each pcb folder (e.g. altana, container) there is a zip file.
+This is what you will add to your pcb order.  Simply download it from this repo and attach to the order using the pcb manufacturer's instructions.
+
+Then, you configure whatever options they need and submit - crucially, the dimensions need to match the actual panel size.
+
+PCB type | panel size
+--- | ---
+Altana | 94x94mm
+Choc | 88.5x84mm
+Container | 94x94mm
+Foostan | ?
```

Comment:
Can you remove this line?
This is footprints folder and foostan is author name.

Reply:
Removed.